en/components/select.md

```diff
@@ -34,20 +35,74 @@ import { IgxSelectModule } from 'igniteui-angular';
+</igx-select>
+```
+
+Alternatively, another way to do it would be to simply pass in a collection of the items that we want to display using [*ngForOf*](https://angular.io/api/common/NgForOf) directive.
+First, specify a collection of the items that you want to display when the drop-down opens:
+```typescript
 public items: string[] = ["Orange", "Apple", "Banana", "Mango"];
 ```
 
 Then in your template you need to bind it with said items like so:
 ```html
-<igx-select>
+<igx-select [(ngModel)]="selected">
+    <igx-select-item *ngFor="let item of items" [value]="item">
+        {{item}}
+    </igx-select-item>
+</igx-select>
+```
+Additionally, we are binding via `ngModel` so there is initial `igxSelect` value set.
+
+We use an `IgxSelectItemComponent` to display the items that `igx-select` operates on. The `IgxSelectItemComponent` extends the [*IgxDropDownItemComponent*]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html) with additional functionality that is specific to the `igx-select-item`.
+
+### Label Prefix and Suffix
+You may have noticed, the `igxSelect` component uses [igx-input-group](input_group.md). This means you can use of it's related directives. Here we are going to utilize `igxLabel`, `igx-prefix` and `igx-suffix`.
+
+```html
+<igx-select [(ngModel)]="selected">
+    <label igxLabel>Pick a fruit</label>
+    <igx-prefix>
+        <igx-icon color="green" style="width:48px; height:32px;">BIO</igx-icon>
+    </igx-prefix>
+    <igx-suffix>
+        <igx-icon (click)="clearSelection($event)">{{ selected ? 'clear' : null }}</igx-icon>
+    </igx-suffix>
     <igx-select-item *ngFor="let item of items" [value]="item">
         {{item}}
     </igx-select-item>
 </igx-select>
 ```
-Notice that we use an `IgxSelectItemComponent` to display the items that `igx-select` operates on. The `IgxSelectItemComponent` extends the [*IgxDropDownItemComponent*](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/classes/igxdropdownitemcomponent.html) with additional functionality that is specific to the `igx-select-item`.
+
+<div class="sample-container loading" style="height: 350px;">
+    <iframe id="select-input-directives-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/data-entries/select-input-directives" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="select-input-directives-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+
+> [!NOTE]
+If no specific `placeholder` is specified for the `igxSelect` component and there is no selection made, the `igxLabel` will transition and appear where you would expect the placeholder to be.
+
+> [!NOTE]
+Currently due to a limitation [`igx-hint`]({environment:angularApiUrl}/classes/igxhintdirective.html) directive can not be used in `igxSelect` component.
```

Comment:
Again, not exactly related to this PR, but is this still a thing? I remember we added `igx-hint` to the combo, perhaps we can enhance the `igx-select` as well?

Reply:
https://github.com/IgniteUI/igniteui-angular/issues/5584